Helper/Data.php

```diff
@@ -19,54 +21,82 @@ class Data extends AbstractHelper
 
     const DHL_API_URL = 'https://api.dhlecommerce.com/';
     const DHL_SANDBOX_API_URL = 'https://api-sandbox.dhlecommerce.com/';
-    const DHL_ENABLE = 'payment/reach_payment/reach_dhl/enable';
-    const DHL_DUTY_LABEL = 'payment/reach_payment/reach_dhl/duty_label';
+    const DHL_ENABLE = 'reach/dhl/enable';
+    const DHL_DUTY_LABEL = 'reach/dhl/duty_label';
     const DHL_DUTY_ALLOW_SPECIFIC = 'reach/dhl/allowspecific';
     const DHL_DUTY_ALLOW_SPECIFIC_COUNTRY = 'reach/dhl/specificcountry';
     const DHL_DUTY_OPTIONAL_SPECIFIC = 'reach/dhl/optional_allowspecific';
     const DHL_DUTY_OPTIONAL_SPECIFIC_COUNTRY = 'reach/dhl/optional_specificcountry';
-    const DHL_DUTY_ALLOW_SHIPPING = 'reach/dhl/applicable_shipping';
-    const DHL_API_KEY = 'reach/dhl/key';
-    const DHL_API_SECRET = 'reach/dhl/api_secret';
-    const DHL_ITEM_SELLER = 'reach/dhl/item_seller';
-    const DHL_PICKUP_ACCOUNT = 'reach/dhl/pickup_account';
-    const DHL_DEFAULT_HS_CODE = 'reach/dhl/default_hs_code';
+
+
+    //DHL_DUTY_ALLOW_SHIPPING would be needed for MAG-90 as well
+    const DHL_DUTY_ALLOW_SHIPPING = "reach/dhl/applicable_shipping";
+    const DHL_API_KEY = "reach/dhl/key";
+    const DHL_API_SECRET = "reach/dhl/api_secret";
+    const DHL_ITEM_SELLER = "reach/dhl/item_seller";
+    const DHL_PICKUP_ACCOUNT =  "reach/dhl/pickup_account";
+    const DHL_DEFAULT_HS_CODE =  "reach/dhl/default_hs_code";
 
     const CONFIG_REACH_ENABLED = 'reach/global/active';
     const CONFIG_CURRENCY_OPTION = 'reach/global/display_currency_switch';
-    const CONFIG_CURRENCY_ALLOWE_SPECIFIC = 'reach/global/allowspecific';
+    const CONFIG_CURRENCY_ALLOW_SPECIFIC = 'reach/global/allowspecific';
     const CONFIG_CURRENCY_SPECIFIC_COUNTRY = 'reach/global/specificcountry';
     const CONFIG_API_MODE = 'reach/global/mode';
     const CONFIG_MERCHANT_ID = 'reach/global/mearchant_id';
     const CONFIG_API_SECRET = 'reach/global/api_secret';
 
-    const CONFIG_CC_OPEN_ONCTRACT = 'payment/reach_cc/allow_open_contract';
 
-    const DHL_PREF_TARIFFS          = "payment/reach_payment/reach_dhl/pref_tariffs";
-    const DHL_PRICING_STRATEGY_PATH = "payment/reach_payment/reach_dhl/pricing_strategy";
-    const DHL_CLEARANCE_MODE_PATH   = "payment/reach_payment/reach_dhl/clearance_mode";
-    const DHL_END_USE_PATH          = "payment/reach_payment/reach_dhl/end_use";
-    const DHL_TRANSPORT_MODE_PATH   = "payment/reach_payment/reach_dhl/transport_mode";
-    const SANDBOX_MODE = 1;
+    const CONFIG_CC_OPEN_CONCTRACT = 'payment/reach_cc/allow_open_contract';
```

Comment:
could you correct this const variable? From CONFIG_CC_OPEN_CONCTRACT to CONFIG_CC_OPEN_CONTRACT

Reply:
Sure thing. And sorry ... my mistake while adding missing C on the pre-existing `CONFIG_CC_OPEN_ONCTRACT`  (on previous line 43 above) I did not pay attention that there were other spelling mistakes there.

Reply:
all good! there are a lot of nitpicky things going on here, I can totally see how that would happen.

Reply:
> The only code change that I would prefer to have done before approving this pull request would be the typo "CONFIG_CC_OPEN_CONCTRACT" to be fixed since that typo was introduced by this code change.
> 
> All other comments were either to learn more about the code changes or make suggestions but I don't see them as required to close this review. Thank you!

Removed commented out debug log from Ryan and corrected the spelling error.
Also added a note (as suggested) indicating a place to correct spelling error in future. All of these are part of commit https://github.com/withreach/magento-plugin/commit/7839c5d4d81a2bda6427cd0af37885e7b62b9a48

 About the other comment: 
The small task of making all the getter methods looking the same by pushing the constants (XML_PATH_ORIGIN_COUNTRY_ID, XML_PATH_ORIGIN_REGION_ID) as defined in the DutyCalculator.php out to Data.php would need adjustment in the code at four different places. Shall handle that as a small ticket when we tackle the batch of tickets to fix pre-existing CC and PayPal code.
Thanks!